Task-20/index.js

```diff
@@ -0,0 +1,60 @@
+class Human {
+    name;
+    age;
+    constructor (name, age) {
+        this.name = name;
+        this.age = age;
+    }
+    information () {
+        console. log(new Human(this.name, this.age))
+    }
+
+}
+
+class Auto {
+    brand;
+    model;
+    year;
+    number;
+    owner;
+
+    constructor (brand, model, year, number) {
+        this.brand = brand;
+        this.model = model;
+        this.year = year;
+        this.number = number;
+    }
+    assignOuner (name) {
```

Comment:
```suggestion
    assignOuner (human) {
```
так буде зрозумілішим, що метод приймає екземпляр класу Людина
(звісно якщо Ви це зробите нижче також треба буде поміняти)